File: SqlOrganize/Sql/EntityRef.cs
namespace SqlOrganize.Sql
{
    public class EntityRef
    {
        public string entityName { get; set; } //tabla foranea que hace referencia a la tabla actual
        public string fieldName { get; set; } //campo fk por el cual se hace referencia al id de la tabla actual
    }
}

File: SqlOrganize/Sql/EntityRef.php
<?php
namespace SqlOrganize\Sql;

class EntityRef
{
    public string $entityName;
    public string $fieldName;

    public static function getInstance($fieldName, $entityName): EntityRef {
        $er = new EntityRef();
        $er->fieldName = $fieldName;
        $er->entityName = $entityName;
        return $er;
    }
}